packages/create-plugin/src/commands/migrate.command.ts

```diff
@@ -57,13 +57,13 @@ export const migrate = async () => {
     // 4. Add / update dependencies inside the `package.json`
     // (skipped automatically if there is nothing to update)
     // ------------------------------------------------
-    if (hasNpmDependenciesToUpdate()) {
+    if (hasNpmDependenciesToUpdate({ dontUpdateGrafanaDependencies: true })) {
```

Comment:
Fixing an (rather extreme edge case) bug where you would get the question to update dependencies but the only to update are grafana dependencies.

For this you would need to have exactly the same dependencies in your old plugin as create-plugin uses. In a way this if will always be true.